src/medCore/gui/commonWidgets/medSlider.cpp

```diff
@@ -0,0 +1,55 @@
+/*=========================================================================
+
+ medInria
+
+ Copyright (c) INRIA 2013 - 2014. All rights reserved.
+ See LICENSE.txt for details.
+ 
+  This software is distributed WITHOUT ANY WARRANTY; without even
+  the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+  PURPOSE.
+
+=========================================================================*/
+
+#include "medSlider.h"
+
+
+medSlider::medSlider() : QSlider()
+{
+}
+
+medSlider::~medSlider()
+{
+}
+
+void medSlider::addTick(int position)
+{
+    if(!ticksList.contains(position))
+        ticksList.append(position);
+}
+void medSlider::removeTick(int position)
+{
+    ticksList.removeAll(position);
+}
+void medSlider::removeAllTicks()
+{
+    ticksList.clear();
+}
+
+void medSlider::paintEvent(QPaintEvent *ev) {
```

Comment:
Since this is a new class, we might as well adopt good habits straight away and keep the same style throughout. Here the opening braces do not follow the style of the rest of the code. Same remark for the braces of the class definition.